src/mdx-pages/guides/tech.mdx

```diff
@@ -108,27 +108,13 @@ Now take the same sources again with videojs-contrib-hls but without videojs-fla
 </video>
 ```
 
-* Safari will play HLS in the html5 tech
-* Chrome will play HLS in the html5 tech by means of videojs-contrib-hls
+* Safari will play HLS in the html5 tech.
+* Chrome will play HLS in the html5 tech.
 * IE 10 can't play HLS in either the html5 or Flash tech. Next the MP4 source will be tested. MP4 can be played by HTML5, so that source-tech pair will be used.
 
-This time, we have videojs-flash but not videojs-contrib-hls:
```

Comment:
My assumption is this section is not longer required as VHS is included by default.

Reply:
Last time this was reviewed it was left in as it was still a useful explanation of how the ordering works, even if not a current example. By now it does just seem completely outdated so fair to remove it.